distributed/client.py

```diff
@@ -468,14 +468,6 @@ def __setstate__(self, state):
         except ValueError:
             c = get_client(address)
         self.__init__(key, c)
-        c._send_to_scheduler(
-            {
-                "op": "update-graph",
-                "tasks": {},
-                "keys": [stringify(self.key)],
-                "client": c.id,
-            }
-        )
```

Comment:
From what I can tell this is not serving any purpose. The future itself lets the scheduler know that it is interested with a `client_desires_keys` so this is redundant and the only non-test invocation of the old API